CHANGELOG.md

```diff
@@ -1430,6 +1438,7 @@ It contains api and sdk for trace and meter.
 - CODEOWNERS file to track owners of this project.
 
 [Unreleased]: https://github.com/open-telemetry/opentelemetry-go/compare/v1.0.0-RC1...HEAD
+[Metrics v0.22.0]: https://github.com/open-telemetry/opentelemetry-go/releases/tag/metrics/v0.22.0
```

Comment:
Not 100% sure how this is going to be tagged, but shouldn't the tag be URL encoded?

```suggestion
[Metrics v0.22.0]: https://github.com/open-telemetry/opentelemetry-go/releases/tag/metrics%2Fv0.22.0
```

Reply:
I don't think so, but thanks for making me check as it should be `metric` and not `metrics`...  See, e.g., https://github.com/open-telemetry/opentelemetry-go/tree/metric/v0.21.0  Looks like URL encoding also works.